### PR TITLE
OSX Arm manual migration

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Basically like #65 except that we drop the c_compiler_version constraint to 10 - it seems to compile fine with the most recent version (12) locally.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
I just discovered https://github.com/conda-forge/texlive-core-feedstock/pull/58 - seems like cross-compiling is not that easy ..

**Comment by ngam** [comment]
> I just discovered #58 - seems like cross-compiling is not that easy ..

Hi @Tobias-Fischer, I worked quite a bit on trying to fix this feedstock. It's rather difficult. Depending on your needs, you may be better off trying a different installation. Currently, we have a lot of errors and basic problems that I personally couldn't figure out a way around. See the issues to get a glimpse...

**Comment by ngam** [comment]
(Sorry I didn't mean to discourage you, more like to warn you...)

**Comment by Tobias-Fischer** [comment]
No, all good. Seems like MacTeX or similar are the way to go at the moment ..
